flow/include/flow/serialize.h

```diff
@@ -887,7 +887,12 @@ class MakeSerializeSource : public ISerializeSource {
 public:
 	using value_type = V;
 	void serializePacketWriter(PacketWriter& w) const override {
-		ObjectWriter writer([&](size_t size) { return w.writeBytes(size); }, AssumeVersion(w.protocolVersion()));
+		ObjectWriter writer(
+		    +[](size_t size, void* pPacketWriter) {
```

Comment:
IIUC, the lambda now does not depend on the template parameters. It could be made a free function (or a static function in a non-template base class), so the cost of template instantiation is reduced further.

Reply:
Good point

Reply:
Really surprised that moving this lambda out saves another 3% of compile time.

Reply:
I am also surprised that it gains _that_ much.